onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -242,6 +243,71 @@ def run_pytorch(use_gpu, model_names, precision, batch_sizes, sequence_lengths,
 
     return results
 
+def run_tensorflow(use_gpu, model_names, precision, batch_sizes, sequence_lengths, repeat_times, cache_dir,
+                   verbose):
+    results = []
+    if use_gpu and not tf.test.is_built_with_cuda():
```

Comment:
Also, when use_gpu is False, we need disable cuda (since tensorflow 2.* has cuda support by default).